lib/fluent/config/types.rb

```diff
@@ -17,7 +17,6 @@
 require 'json'
 
 require 'fluent/config/error'
-require 'fluent/configurable'
```

Comment:
It's better to:
* remove this requirement
* move `Configurable.register_type` calls to `configurable.rb`
* move `require 'fluent/config/types'` at the end of `configurable.rb` to head of that file